app/Http/Controllers/Api/AssetModelsController.php

```diff
@@ -127,6 +128,7 @@ public function update(Request $request, $id)
         $this->authorize('edit', AssetModel::class);
         $assetmodel = AssetModel::findOrFail($id);
         $assetmodel->fill($request->all());
+        $assetmodel->fieldset_id = $request->get("custom_fieldset_id");
```

Comment:
Please use $request->input('foo') or request('foo') rather than $request->get, Request::get is a symfony thing that can provide some weird results.

Also--Is it better to make the fieldset_id fillable rather than having to manually add it? That way it happens as part of Model::fill()